sample/src/main/java/com/chuckerteam/chucker/sample/SampleApiService.java

```diff
@@ -50,53 +61,103 @@ static HttpbinApi getInstance(OkHttpClient client) {
     interface HttpbinApi {
         @GET("/get")
         Call<Void> get();
+
         @POST("/post")
         Call<Void> post(@Body Data body);
+
         @PATCH("/patch")
         Call<Void> patch(@Body Data body);
+
         @PUT("/put")
         Call<Void> put(@Body Data body);
+
         @DELETE("/delete")
         Call<Void> delete();
+
         @GET("/status/{code}")
         Call<Void> status(@Path("code") int code);
+
         @GET("/stream/{lines}")
         Call<Void> stream(@Path("lines") int lines);
+
         @GET("/stream-bytes/{bytes}")
         Call<Void> streamBytes(@Path("bytes") int bytes);
+
         @GET("/delay/{seconds}")
         Call<Void> delay(@Path("seconds") int seconds);
+
         @GET("/bearer")
         Call<Void> bearer(@Header("Authorization") String token);
+
         @GET("/redirect-to")
         Call<Void> redirectTo(@Query("url") String url);
+
         @GET("/redirect/{times}")
         Call<Void> redirect(@Path("times") int times);
+
         @GET("/relative-redirect/{times}")
         Call<Void> redirectRelative(@Path("times") int times);
+
         @GET("/absolute-redirect/{times}")
         Call<Void> redirectAbsolute(@Path("times") int times);
+
         @GET("/image")
         Call<Void> image(@Header("Accept") String accept);
+
         @GET("/gzip")
         Call<Void> gzip();
+
         @GET("/xml")
         Call<Void> xml();
+
         @GET("/encoding/utf8")
         Call<Void> utf8();
+
         @GET("/deflate")
         Call<Void> deflate();
+
         @GET("/cookies/set")
         Call<Void> cookieSet(@Query("k1") String value);
+
         @GET("/basic-auth/{user}/{passwd}")
         Call<Void> basicAuth(@Path("user") String user, @Path("passwd") String passwd);
+
         @GET("/drip")
         Call<Void> drip(@Query("numbytes") int bytes, @Query("duration") int seconds, @Query("delay") int delay, @Query("code") int code);
+
         @GET("/deny")
         Call<Void> deny();
+
         @GET("/cache")
         Call<Void> cache(@Header("If-Modified-Since") String ifModifiedSince);
+
         @GET("/cache/{seconds}")
         Call<Void> cache(@Path("seconds") int seconds);
     }
+
+    interface GraphQLSWApi {
```

Comment:
Can you move this to a separate class, like a `SimpleGraphQLService` or something similar